src/actions/commands/actions.ts

```diff
@@ -2145,7 +2145,7 @@ export class CommandYankFullLine extends BaseCommand {
   public async exec(position: Position, vimState: VimState): Promise<VimState> {
     const linesDown = (vimState.recordedState.count || 1) - 1;
     const start = position.getLineBegin();
-    const end = new Position(position.line + linesDown, 0).getLineEnd().getLeft();
+    const end = position.getDownByCount(linesDown).getLeft();
```

Comment:
Why can we drop the `getLineEnd()`?

Reply:
I thought it is unnecessary because end position is recalculated in the `YankOperator#run`.

```typescript
    let extendedEnd = new Position(end.line, end.character + 1);

    if (vimState.currentRegisterMode === RegisterMode.LineWise) {
      start = start.getLineBegin();
      extendedEnd = extendedEnd.getLineEnd();
    }
```
(operator.ts L284)

However, if so, `getLeft` also seems unnecessary...